uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,32 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+
+  return page.extend({
+    /**
+     * [init page]
```

Comment:
Other files had [], and I followed that. Later found that that [] was present as la-milestone-3 is not synced with 1.6. Removed the [].